fflas-ffpack/fflas/fflas_fgemm.inl

```diff
@@ -410,7 +410,9 @@ namespace FFLAS {
             else if (!std::is_same<Field,Givaro::ModularBalanced<float> >::value){
                 if (F.characteristic() < DOUBLE_TO_FLOAT_CROSSOVER)
                     return Protected::fgemm_convert<Givaro::ModularBalanced<float>,Field>(F,ta,tb,m,n,k,alpha,A,lda,B,ldb,beta,C,ldc,H);
-                else if (!std::is_same<Field,Givaro::ModularBalanced<double> >::value && 16*F.cardinality() < Givaro::ModularBalanced<double>::maxCardinality())
+                else if (!std::is_same<Field,Givaro::ModularBalanced<double> >::value &&
+			 !std::is_same<Field,Givaro::ModularBalanced<double> >::value &&
```

Comment:
why is this line duplicated?

Reply:
If I remember well, we did this together to correct some compile time error, but I could not remember of the exact reason.